app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+    @post = Post.find(params[:post_id])
+    @user = @current_user
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @user = User.find(params[:user_id])
+    @comment = @post.comments.new(comment_params.merge(author: @user))
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/forms_project_v1_1.md#:~:text=Create%20a%20comment%20on%20behalf%20of%20the%20current_user), Please create a comment on behalf of the current_user (use the method that you created in your ApplicationController).

[Hint] You can make it like that:

```ruby
def create
    @comment = @current_user.comments.new(params)

    # The rest of the code here
```